### PR TITLE
Event slot selector refactor

### PR DESCRIPTION
**Pull request by oliveiraigorm**

### CODE REVIEW

File: src/components/Shared/CalendarPicker.vue [hunk 1]
```diff
@@ -96,4 +98,26 @@ export default {
 .calendar-picker__tag + .calendar-picker__tag {
 	margin-left: 5px;
 }
+
+.wrapper {
+	display: flex;
+	justify-content: space-around;
+}
+
+ul {
```
Comment (mwalbeck) [inline src/components/Shared/CalendarPicker.vue:107]: Is this rule being used for anything?

File: src/components/Shared/CalendarPicker.vue [hunk 2]
```diff
@@ -96,4 +98,26 @@ export default {
...
+	display: flex;
+	justify-content: space-around;
+}
+
+li {
```
Comment (mwalbeck) [inline src/components/Shared/CalendarPicker.vue:112]: Is this rule being used for anything?

**Review by mwalbeck — COMMENTED** [review]
Two quick questions, the rest is looking good!

**Comment by mwalbeck** [comment]
One more thing, could you add the `cursor: pointer` css rule to the radio buttons so the cursor behaves more like one would expect?

File: src/components/Shared/CalendarPicker.vue (hunk 1 above)
Comment (oliveiraigorm) [reply src/components/Shared/CalendarPicker.vue]: nope, I've forgot to remove them

**Review by mwalbeck — APPROVED** [review]
Nice! Good job!

**Pull request merged**